### PR TITLE
feat(config): add MCO Home MH5-2a and MH-S510, correct others

### PR DESCRIPTION
**Pull request by blhoward2**
Adds new device and merges in ZWA changes to MCO Home files

### CODE REVIEW

**Comment by blhoward2** [comment]
@robertsLando I need the device type/id. I didn't see those in the manual so can't merge this.

**Comment by robertsLando** [comment]
Thanks so much @blhoward2 ! Will give you all the info as soon as I receive the device, maybe this or the next week.

**Comment by robertsLando** [comment]
BTW There could be other 3 models that have a quite similar configuration but I don't own manuals of them: 

Schermata da 2021-08-02 09-57-44

**Comment by AlCalzone** [comment]
We could also email their support. Last time I had contact they were somewhat responsive.

**Comment by robertsLando** [comment]
Yeah the support is very responsive. I can ask them if you want or do you already have a contact?

**Comment by AlCalzone** [comment]
You do it :)

**Comment by robertsLando** [comment]
Waiting their respoonse, email sent :)

**Comment by robertsLando** [comment]
Just received the pack. I have to say I really like the design! It's half the price of the heatit one but much more robust

image

image

**Comment by robertsLando** [comment]
Here are the informations you requested: https://devices.zwave-js.io/?jumpTo=0x015f:0xf501:0x7102:0.0

[36-114-0-manufacturerId] Manufacturer ID
351
[36-114-0-productType] Product type
62721
[36-114-0-productId] Product ID
28930

**Comment by blhoward2** [comment]
I'm out of town for a few days. @AlCalzone can you add it and merge?

File: packages/config/config/devices/0x015f/mh-s510.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH-S510",
+	"description": "Touch Panel Switch",
+	"devices": [
+		{
+			"productType": "0x5131",
+			"productId": "0x5101",
```
Comment (robertsLando) [inline packages/config/config/devices/0x015f/mh-s510.json:9]:
```suggestion
			"productType": "0xf501",
			"productId": "0x7102",
```

**Comment by robertsLando** [comment]
Cool thing I just noticed it also has an indicator that shows how good is the signal, like we have in smartphones. It's the first device I see with this :)

File: packages/config/config/devices/0x015f/mh-s510.json (hunk 1 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x015f/mh-s510.json:9]: Is that from the support? Or did you mean to edit this?
https://github.com/zwave-js/node-zwave-js/pull/3124/files#diff-8917909cccb9b91d0b1261fdfe51094786376bec6cb0b69736176022b50b0d58R8

Comment (robertsLando) [reply packages/config/config/devices/0x015f/mh-s510.json:9]: Them are the values reported by the device

Comment (robertsLando) [reply packages/config/config/devices/0x015f/mh-s510.json:9]: @blhoward2 Needed them to finish the PR: https://github.com/zwave-js/node-zwave-js/pull/3124#issuecomment-890279622

Comment (AlCalzone) [reply packages/config/config/devices/0x015f/mh-s510.json:9]: Yeah, but not in the file for the S510?!

Comment (robertsLando) [reply packages/config/config/devices/0x015f/mh-s510.json:9]: Ops, sorry that was the wrong file, fixing now

File: packages/config/config/devices/0x015f/mh5-2a.json [hunk 2]
```diff
@@ -0,0 +1,389 @@
+{
+	"manufacturer": "McoHome Technology Co., Ltd.",
+	"manufacturerId": "0x015f",
+	"label": "MH5-2A",
+	"description": "Fan Coil Thermostat",
+	"devices": [
+		{
+			"productType": "Need",
+			"productId": "Need"
```
Comment (robertsLando) [inline packages/config/config/devices/0x015f/mh5-2a.json]:
```suggestion
			"productType": "0xf501",
			"productId": "0x7102",
```

**Review by robertsLando — COMMENTED** [review]
@AlCalzone

File: packages/config/config/devices/0x015f/mh5-2a.json [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 0
```
Comment (AlCalzone) [inline packages/config/config/devices/0x015f/mh5-2a.json]: @blhoward2 this is supposed to be 1, right?

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x015f/mh5-2a.json (hunk 3 above)
Comment (robertsLando) [reply packages/config/config/devices/0x015f/mh5-2a.json]: I confirm this should be 1 yeah, based on manual

File: packages/config/config/devices/0x015f/mh5-2a.json [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+				}
+			]
+		},
+		"2": {
+			"label": "Temperature Reporting Threshold",
+			"description": "Valid range: 3-255; Variation in temperature to induce automatic report",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 255,
```
Comment (robertsLando) [inline packages/config/config/devices/0x015f/mh5-2a.json:48]:
```suggestion
			"maxValue": 255,
			"unit": "0.1 °F/C",
```

File: packages/config/config/devices/0x015f/mh5-2a.json (hunk 3 above)
Comment (blhoward2) [reply packages/config/config/devices/0x015f/mh5-2a.json]: I'm fine if you want to make Celsius the default.

Comment (AlCalzone) [reply packages/config/config/devices/0x015f/mh5-2a.json]: I meant the option value, we have zero twice.

**Comment by AlCalzone** [comment]
Yo @zwave-js-bot what's up? Not commenting?
@zwave-js-bot fix lint!

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**